src/inc/WindowingBehavior.h

```diff
@@ -0,0 +1,10 @@
+/*++
+Copyright (c) Microsoft Corporation
+Licensed under the MIT license.
+--*/
+
+#pragma once
+constexpr int64_t WindowingBehaviorUseCurrent{ 0 };

```

Comment:
do we have a `cascadia/inc`?

Reply:
this seems like a contract between remoting and apphost honestly

Reply:
okay sure;

Yea this one's awkward because TerminalApp definitely needs to be aware of these values for `FindTargetWindow` to work right, but App doesn't otherwise depend on remoting at all. Plus, we can't just have `FindTargetWindow` return a `WindowingMode`, because it technically needs to return a `union{ WindowingMode, uint }`, so ¯\\\_(ツ)_/¯